pages/common/zoxide.md

```diff
@@ -0,0 +1,28 @@
+# zoxide
+
+> Keep track of the directories you use most frequently, and use a ranking algorithm to navigate to the best match.
+> More information: <https://github.com/ajeetdsouza/zoxide>.
+
+- Go to the highest-ranked directory that contains "foo" in the name:
+
+`zoxide query {{foo}}`
+
+- Go to the highest-ranked directory that contains "foo" and then "bar":
+
+`zoxide query {{foo}} {{bar}}`
+
+- Interactive directory search (requires [fzf](https://github.com/junegunn/fzf)):
```

Comment:
Perhaps we could tweak this to make it more consistent with other pages? We already have a page for `fzf`, so we can reference it here.
```suggestion
- Start an interactive directory search (requires `fzf`):
```